ListDetail/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="list_details_app_name">Listendetail</string>
```

Comment:
It can work, but as before, we should go for an unified approach for sample-names